Makefile

```diff
@@ -12,17 +13,28 @@ all:
 
 sanity:
 	@for file in $$(find lib -type f -name "*.pm" | sort); do \
-		perl -cw -Ilib $$file || exit 1; \
+		$(PERL_BIN)/perl -cw -Ilib $$file || exit 1; \
 	done
 
 test: sanity
 	-$(MAKE) elevate-cpanel
-	/usr/local/cpanel/3rdparty/bin/prove t/00_load.t
-	/usr/local/cpanel/3rdparty/bin/yath test -j8 t/*.t
+	$(PERL_BIN)/prove t/00_load.t
+	$(PERL_BIN)/yath test -j8 t/*.t
+
+build:
+    curl -fsSL https://raw.githubusercontent.com/skaji/cpm/main/cpm > /usr/sbin/cpm
```

Comment:
might be better off copying into the repo. it doesn't need to be in sbin.

Reply:
OK added it in and then `make clean` will remove the file.